### PR TITLE
some stuff

### PR DESCRIPTION
**Pull request by Yugenero**
title

### CODE REVIEW

**Review by allisonfain08 — APPROVED** [review]
looks really good. should be able to overwrite the .DS_Store error that is preventing from merging.

File: rnFirebaseAuth/App.js [hunk 1]
```diff
@@ -49,4 +49,6 @@ export default function App() {
       <MyStack />
     </NavigationContainer>
   );
-}
\ No newline at end of file
+}
+
+// figure out with shonjoy about how to be able to implement the swiping feature using images from tanjuria
```
[the remark this answers is not in the document]
Comment (ShonjoyBhattacharyya) [reply rnFirebaseAuth/App.js:54]: This is correct, xcode has a built in swipe gesture recognizer that will be useful for us

**Review by MijaKennedy — APPROVED** [review]
comments look good in line, changes seem fine

**Review by olivia-mauger — COMMENTED** [review]
Great updates to read-me, no changes to functionality of code.

**Pull request closed without merging**